core/services/relay/delegate_test.go

```diff
@@ -68,3 +65,19 @@ func TestNewOCR2Provider(t *testing.T) {
 		})
 	}
 }
+
+var _ terra.ChainSet = (*mockChainSet)(nil)
+
+type mockChainSet struct{}
+
+func (m mockChainSet) Start() error { return nil }
+
+func (m mockChainSet) Close() error { return nil }
+
+func (m mockChainSet) Ready() error { return nil }
+
+func (m mockChainSet) Healthy() error { return nil }
+
+func (m mockChainSet) Get(id string) (terra.Chain, error) {
+	return nil, errors.New("this is a mock ChainSet")
+}
```

Comment:
why not just generate this?

Reply:
The interface is in the `chainlink-terra` repo. I'll try again to rig up mockery though.

Reply:
ohh I see. I assume you can still generate mocks for imported interfaces. If not, it's nbd.

Reply:
`mockery --srcpkg` :heavy_check_mark: